packages/atomic/src/components/atomic-query-error/atomic-query-error.tsx

```diff
@@ -129,19 +145,24 @@ export class AtomicQueryError implements InitializableComponent {
 
     return (
       <div class="text-center">
-        <h3 part="title" class="text-2xl text-primary my-4">
+        <atomic-icon
+          part="icon"
+          icon={this.icon}
+          class="w-1/2 mt-8"
+        ></atomic-icon>
```

Comment:
`atomic-icon` is only meant to display 1:1 icons, did you test this with `indexing.svg` and `no-connection.svg`?